tests/test_parallel_post_fit.py

```diff
@@ -49,6 +51,22 @@ def test_laziness():
     assert 0 < x.compute() < 1
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
+def test_predict_with_meta():
```

Comment:
Can you also add a test asserting that a warning is raised when `meta` isn't provided?


```python
with pytest.warns(FutureWarning):
    ....
```

Reply:
Thanks a lot for the review. Will add  that.

Reply:
Added test here: https://github.com/dask/dask-ml/blob/094c31adc3352c50966f86d501739b1ea50a9c9e/tests/test_parallel_post_fit.py#L60-L66